js/tiles.js

```diff
@@ -1716,7 +1716,7 @@ const marvel = {
 
       visit: function() {
 
-        if (player.inventory.has('Soul') ) {
+        if (true) {
         player.addReachableLevel(createFirstLevel());
        } else {
            alertInfo("Get all the six infinty stones to advance to the next level...");
```

Comment:
why is there an else associated with always true statement?

Reply:
This statement won't be true always